rollup.config.js

```diff
@@ -16,6 +17,9 @@ function generateConfig(configType, format) {
     input: 'src/index.ts',
     plugins: [
       commonjs(),
+      nodePolyfills({
+        include: ['events']
```

Comment:
Do we need to add `stream` to this list?

Reply:
@kklash I haven't adequately tested the changes yet. After adding `events`, the error was gone, so I did not add `stream`. I suggest that we hold off with merging this PR until I finish testing.

Reply:
Added do-not-merge tag.

Reply:
Removed the `options` arg altogether to fix the below error:

```
src/index.ts → lib/index.cjs.js, lib/index.esm.js...
(!) Plugin node-polyfills: rollup-plugin-inject: failed to parse /Users/nishant/Projects/solana-web3.js/src/account.ts. Consider restricting the plugin to particular files via options.include
src/account.ts
(!) Plugin node-polyfills: rollup-plugin-inject: failed to parse /Users/nishant/Projects/solana-web3.js/src/nonce-account.ts. Consider restricting the plugin to particular files via options.include
src/nonce-account.ts
(!) Plugin node-polyfills: rollup-plugin-inject: failed to parse /Users/nishant/Projects/solana-web3.js/src/vote-account.ts. Consider restricting the plugin to particular files via options.include
src/vote-account.ts
(!) Plugin node-polyfills: rollup-plugin-inject: failed to parse /Users/nishant/Projects/solana-web3.js/src/ed25519-program.ts. Consider restricting the plugin to particular files via options.include
src/ed25519-program.ts
(!) Plugin node-polyfills: rollup-plugin-inject: failed to parse /Users/nishant/Projects/solana-web3.js/src/fee-calculator.ts. Consider restricting the plugin to particular files via options.include
src/fee-calculator.ts
```